workers/reporter/src/main/kotlin/reporter/ReportNameMapper.kt

```diff
@@ -77,7 +77,7 @@ interface ReportNameMapper {
     }
 
     /**
-     * Return a map that assigns each element of the given collection of [reportFiles] a name. The report file is then
+     * Return a map that assigns each element of the given [reportFiles] collection by a name. The report file is then
```

Comment:
"assigns ... by a name" doesn't work, better: "...assigns a name to each element of..."

Reply:
I believe "assigns *each element* ... by a name" *does* work. I'll change it nonetheless.